src/three_by_three_validator.py

```diff
@@ -1,32 +1,20 @@
 import src.constants as constants
 
 class ThreeByThreeValidator():
-    def __init__(self, validator_type):
-        self.type = validator_type
+    def __init__(self, board):
+        self.board = board
 
-    def validate_move(self, move, board):
-        if board.type == constants.THREE_BY_THREE:
-            return self.validate_3x3(move, board)
-
-    def validate_3x3(self, move, board):
-        if move.lower() == constants.EXIT:
-            return [constants.EXIT, None]
-        elif self.validate_3x3_input(move, board) and self.validate_3x3_move(move, board):
-             return [constants.MOVE, move] 
+    def validate(self, move):
+        move = move.lower()
+        if self.validate_input(move):
+            if move == constants.EXIT:
+                return [constants.EXIT, None]
+            elif self.board.valid_move(move):
+                return [constants.MOVE, move]
```

Comment:
If you can avoid nested conditionals by abstracting some of this into private methods it can be helpful. one nesting isn't so bad, but it can create a pattern that when followed by future devs can very quickly get tangled.